### PR TITLE
Add game page

### LINKED ISSUE #39: Create view to show select field and call use case shot

### PR DESCRIPTION
**Pull request by iurynogueira**
fix #39

### CODE REVIEW

**Review by MarcosOliveir4 — APPROVED** [review]

File: src/app/pages/game/game.component.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<div class="container">
+  <h2>Tente adivinhar o campeão selecionando um capeão abaixo.</h2>
```
Comment (mizek1) [inline src/app/pages/game/game.component.html:2]:
```suggestion
  <h2>Tente adivinhar o campeão selecionando um campeão abaixo.</h2>
```

**Review by mizek1 — CHANGES_REQUESTED** [review]

**Comment by iurynogueira** [comment]
> This is the final layout? I think we should use icons instead of text to sinalize the stats and if they're bigger, smaller etc. I also think that the user should be able to type the name of a champion and then select it from the list, not only select from the list.

No. We can improve this result layout now, in select we can open other issue to improve her, what do you think?

**Comment by mizek1** [comment]
Hmmmm I just noticed, we forgot to add the champion class comparison, right? (If it's fighter, assassin, etc.)

**Comment by iurynogueira** [comment]
> Hmmmm I just noticed, we forgot to add the champion class comparison, right? (If it's fighter, assassin, etc.)

Sorry, added and improve layout to correct icons.

image

**Review by mizek1 — APPROVED** [review]

**Pull request merged**